app/views/layouts/_application_footer.html.erb

```diff
@@ -11,7 +11,9 @@
     </div>
 
     <div id="footer-copyright">
-      <%= link_to OpenStax::Utilities::Text.copyright('2013', COPYRIGHT_HOLDER), main_app.copyright_path %>
+      <%= link_to(
+        OpenStax::Utilities::Text.copyright('2013', COPYRIGHT_HOLDER), main_app.copyright_url
```

Comment:
Hard coded year?

Reply:
This one makes text like `Copyright &copy; 2013-2019 Rice University` so only the starting year is hard coded which makes sense to me.

Reply:
Oh, so someone coded up the 're-assert copyright every year' statement.  Ok. Not sure if it has any impact legally, if there's no changes in the current year, but this is a dynamic web page, anyway.